experiments/pages/childNormalPageConverter/childNormalPageConverter_Salesforce.xml

```diff
@@ -0,0 +1,521 @@
+      <fields uniqueId="sk-3dKp-1472">
+        <field id="skuid__Layout__c"/>
+        <field id="skuid__Layout2__c"/>
+        <field id="skuid__Layout3__c"/>
+        <field id="skuid__Layout4__c"/>
+        <field id="skuid__Layout5__c"/>
+        <field id="Id"/>
+        <field id="skuid__API_Version__c"/>
+        <field id="skuid__MasterPage__c"/>
+        <field id="skuid__MasterPage__r.Name"/>
+        <field id="Name"/>
+      </fields>
+      <idfields>
+        <field>id</field>
+      </idfields>
+      <conditions logic="" uniqueId="sk-3dKp-1484">
+        <condition type="fieldvalue" value="" enclosevalueinquotes="true" field="Id" state="filterableon" inactive="false" name="Id"/>
+        <condition type="fieldvalue" value="v1" enclosevalueinquotes="true" field="skuid__API_Version__c"/>
+      </conditions>
+      <actions uniqueId="sk-3dKp-1485"/>
+    </model>
+    <model id="masterPages_retrieveXML" query="false" processonclient="true" type="readwrite" datasource="salesforce" label="Page" labelplural="Pages" namefield="name" uniqueId="sk-3dKp-1466" sobject="skuid__Page__c" limit="1">
+      <fields uniqueId="sk-3dKp-1472">
+        <field id="skuid__Layout__c"/>
+        <field id="skuid__Layout2__c"/>
+        <field id="skuid__Layout3__c"/>
+        <field id="skuid__Layout4__c"/>
+        <field id="skuid__Layout5__c"/>
+        <field id="Id"/>
+        <field uionly="true" displaytype="FORMULA" ogdisplaytype="TEXT" readonly="true" returntype="TEXT" id="fullXML" label="Full XML">
+          <formula>{{skuid__Layout__c}} +
+{{skuid__Layout2__c}} +
+{{skuid__Layout3__c}} +
+{{skuid__Layout4__c}} +
+{{skuid__Layout5__c}}</formula>
+        </field>
+      </fields>
+      <idfields>
+        <field>id</field>
+      </idfields>
+      <conditions logic="" uniqueId="sk-3dKp-1484">
+        <condition type="fieldvalue" value="" enclosevalueinquotes="true" field="Id" state="filterableon" inactive="false" name="Id"/>
+        <condition type="fieldvalue" value="v1" enclosevalueinquotes="true" field="skuid__API_Version__c"/>
+      </conditions>
+      <actions uniqueId="sk-3dKp-1485"/>
+    </model>
+  </models>
+  <components>
+    <tabset rememberlastusertab="true" defertabrendering="true" uniqueid="sk-3Px3-490">
+      <tabs>
+        <tab name="Pages List">
+          <components>
+            <skootable showconditions="true" showsavecancel="true" showerrorsinline="true" searchmethod="server" searchbox="true" showexportbuttons="false" hideheader="false" hidefooter="false" pagesize="10" alwaysresetpagination="false" createrecords="true" model="pages" buttonposition="" mode="read" allowcolumnreordering="true" responsive="true" uniqueid="sk-NbK-453">
+              <fields>
+                <field id="skuid__API_Version__c" uniqueid="fi-NbK-454"/>
+                <field id="Name" hideable="true" uniqueid="fi-Neo-1934"/>
```

Comment:
To prevent people from being redirected to Page Composer wrapped in Lightning header:

```suggestion
                <field id="Name" hideable="true" uniqueid="fi-Neo-1934" redirecttype="customurl" window="self" rooturl="/apex/skuid__PageBuilder?id={{Id}}"/>
```